commands/transform.ts

```diff
@@ -1,23 +1,22 @@
 import { join } from 'node:path'
-import execa from 'execa'
```

Comment:
Is this still in use? Should we keep it in package json file?

Reply:
I might be able to use it in the update command, but if you want, I can remove it.